sunset/code/game/objects/structures/beds_chairs/chair.dm

```diff
@@ -0,0 +1,50 @@
+/obj/structure/chair/wood
+	icon = 'sunset/icons/obj/chairs.dmi'
+
+/obj/structure/chair/comfy
+	icon = 'sunset/icons/obj/chairs.dmi'
+
+/obj/structure/chair/comfy/proc/GetArmrest()
```

Comment:
instead of having to comment out code just dont put /proc/

```suggestion
/obj/structure/chair/comfy/GetArmrest()
```

Reply:
Tested it, gave me a error about those procs being undefined.